webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -29,106 +114,395 @@ def _main_handle(self, stored_data):
         return stored_data
 
 
-class EligibilityIncomesFormSteuerlotseStep(FormSteuerlotseStep):
-    name = 'incomes'
-    title = _l('form.eligibility.income-title')
-    intro = _l('form.eligibility.income-intro')
-    template = 'eligibility/form_incomes.html'
+class MaritalStatusInputFormSteuerlotseStep(EligibilityInputFormSteuerlotseStep):
+    name = "marital_status"
+    title = _('form.eligibility.marital_status-title')
+    intro = _('form.eligibility.marital_status-intro')
+    next_steps = {
+        'married': "separated",
+        'widowed': "single_alimony",
+        'single': "single_alimony",
+        'divorced': "divorced_joint_taxes",
+    }
```

Comment:
Let's talk about it again, because I think the steps changed a little. That said, we could also use a dict but this would make it a little less obvious that we are only deciding between two different steps

Reply:
s. other answer about marital status input step